test/cs_undefined/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Boost.Geometry
+# Copyright (c) 2024, Oracle and/or its affiliates.
+# Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+# Use, modification and distribution is subject to the Boost Software License,
+# Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+foreach(item IN ITEMS
+    distance
+    envelope_expand
+    index
+    is
+    measure
+    other
+    relops1
+    relops2
+    setops1
+    setops2
+  )
+  boost_geometry_add_unit_test("cs_undefined" ${item})
+endforeach()
```

Comment:
So, in line with my comment above, you would get a second loop here

```
foreach(item IN ITEMS
    setops1
    setops2
  )
  boost_geometry_add_unit_test("cs_undefined" ${item} FALSE)
endforeach()
```

Reply:
Thanks, I added then one by one though.